src/components/Cloud/Pull-Request-Item.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Display status about a single Pull Request.
+ *
+ * @param {String} date       Date that the pull request was opened.
+ * @param {Number} id         GitHub Pull Request identifier.
+ * @param {String} link       Link to the pull request on GitHub.
+ * @param {String} status     Un-mapped status of the pull request
+ * @param {String} statusText Formatted string describing the current status of the pull request,
+ * @param {String} title      Title of the pull request.
+ */
+const PullRequestItem = ( { date, id, link, status, statusText, title } ) => {
+	const parsedDate = new Date( date );
+	return <li className="pull-request-item">
+		<div className="pull-request-item__info" >
+			<p className="pull-request-item__title">
+				<a href={ link }>#{ id }</a> <strong>{ title }</strong>
+			</p>
+			<p className="pull-request-item__date"><i>{ parsedDate.toLocaleDateString() }</i></p>
```

Comment:
We could use more semantic markup here. `<i>` is purely a stylistic tag so `<em>` would be better but `<time>` would be better still. That said I've not looked at the CSS yet and whether this is to leverage WP admin styles.